### PR TITLE
feat: open a flow

### PR DESCRIPTION
**Pull request by mshanemc**
### What does this PR do?
org:open --source-file can open a flow in flow builder

### What issues does this PR fix or reference?
https://github.com/forcedotcom/cli/discussions/2519
@W-14287109@

### CODE REVIEW

File: messages/open.md [hunk 1]
```diff
@@ -67,3 +71,12 @@ Waiting to resolve the Lightning Experience-enabled custom domain...
 # domainTimeoutError
 
 The Lightning Experience-enabled custom domain is unavailable.
+
+# FlowIdNotFound
+
+No ID not found for Flow %s .
```
Comment (WillieRuemmele) [inline messages/open.md:77]: NIT: we should move the `.` to be at the end of the word, not ` .`

File: test/nut/open.nut.ts [hunk 2]
```diff
@@ -65,6 +66,15 @@ describe('test org:open command', () => {
     expect(result.url).to.include('/visualEditor/appBuilder.app?pageId');
   });
 
+  it('should produce the URL for an existing flow', () => {
+    const result = execCmd<OrgOpenOutput>(`force:source:open --source-file ${flowPath} --url-only --json`, {
```
Comment (WillieRuemmele) [inline test/nut/open.nut.ts:70]: any reason to be using the deprecated command here?

Comment (mshanemc) [reply test/nut/open.nut.ts:70]: nope, just that the others were.  A mix is probably best.

**Review by WillieRuemmele — APPROVED** [review]

**Comment by WillieRuemmele** [comment]
### QA Notes
---

### setup
create a flow, and pull it into the project

`sf org open --source-file force-app/main/default/flows/my_flow-meta.xml` => 
image

**Pull request merged**